ui/warrior/inputs.ts

```diff
@@ -299,13 +294,10 @@ export const WarriorRotationConfig = {
 			label: 'HS rage threshold',
 			labelTooltip: 'Queue HS when rage is above:',
 			changedEvent: (player: Player<Spec.SpecWarrior>) => player.rotationChangeEmitter,
-			getValue: (player: Player<Spec.SpecWarrior>) => player.getRotation().general?.hsRageThresh || 60,
+			getValue: (player: Player<Spec.SpecWarrior>) => player.getRotation().hsRageThreshold || 70,
```

Comment:
When any of these values is missing, its important to return a zero/false/empty value or there will be a desync between the visuals and the internal representation. Actual default values are handled by your Presets.DefaultXXX fields.